tests/utils_tests/test_lorem_ipsum.py

```diff
@@ -1,14 +1,73 @@
 import unittest
 
-from django.utils.lorem_ipsum import paragraphs, words
+from django.utils.lorem_ipsum import paragraph, paragraphs, sentence, words
 
 
 class WebdesignTest(unittest.TestCase):
 
-    def test_words(self):
+    def test_negative_words(self):
+        """
+        Common words when passing negative words
```

Comment:
Docstrings should generally try to state the expected behavior, be a single line if they are short enough (within the 79 character limit), and end with a period.